src/parameter_declarations.ts

```diff
@@ -243,6 +249,31 @@ Parameter.declare({
 	type: 'boolean'
 });
 
+Parameter.declare({
+	name: SkipParameter,
+	aliases: ['--skip'],
+	description: '(OPTIONAL) Skip upload if there is a name conflict within destination folder',
+	type: 'boolean',
+	forbiddenConjunctionParameters: [ReplaceParameter, UpsertParameter]
+});
+
+Parameter.declare({
+	name: ReplaceParameter,
+	aliases: ['--replace'],
+	description: '(OPTIONAL) Create new file revisions if there is a name conflict within destination folder',
+	type: 'boolean',
+	forbiddenConjunctionParameters: [SkipParameter, UpsertParameter]
+});
+
+Parameter.declare({
+	name: UpsertParameter,
+	aliases: ['--upsert'],
+	description:
+		'(OPTIONAL) When there is a name conflict within the destination folder, if that file was last modified at the same time as the file to upload, skip the upload, otherwise upload that file as a new revision',
```

Comment:
Can we generalize the language here to consider that this could be applied to file or folder uploads?

Reply:
Actually, upsert logic doesn't really effect folder conditions. As per the [table](https://github.com/ardriveapp/ardrive-cli/tree/dev#name-conflict-resolution-on-upload), folder to X conflicts shares the behaviors of replace. And the only difference it with skip is that it will `Fail` instead of skipping.

This message should be simplified though, how about:

```suggestion
		"(OPTIONAL) Update files only when they've been modified if there is a name conflict within the destination folder",
```

Reply:
The language is a little clunky. Reorder to:
```
(OPTIONAL)  When there is a name conflict within the destination folder, only upload file if a modification is detected. Skip otherwise.
```

Reply:
9fff5cc